app/__init__.py

```diff
@@ -59,15 +60,15 @@
     app, allow_headers=["X-Requested-With"], allow_origin=app.config["ALLOW_ORIGIN"],
 )
 
-if "pytest" in sys.modules:
-    from simplekv.memory import DictStore
-
-    store = DictStore()
-else:
-    store = RedisStore(redis.StrictRedis(host=app.config["REDIS_HOST"]))
+if "pytest" not in sys.modules:
+    # Set up the redis store for tokens
+    app.store = OAuthRedis(
+        fernet_key=app.config["FERNET_KEY"], host=app.config["REDIS_HOST"]
```

Comment:
Should we add `FERNET_KEY` and `REDIS_HOST` to `README.rst` as well?

Reply:
`REDIS_HOST ` is already there. 
`FERNET_KEY` (32 bytes) was created from `SECRET_KEY`(also 32 bytes, stored as hex) at startup. I moved this conversion to the init of the OAuthRedis class which is cleaner.
For the adoption of dotenv I've created an issue: https://github.com/SwissDataScienceCenter/renku-gateway/issues/235